.travis.yml

```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: ruby
 rvm:
-- 2.3.1
+  - 2.4.1
```

Comment:
I'm not an experienced Ruby programmer – does bumping to 2.4.x make it difficult for us to verify compatibility with 2.3.x?